### ISSUE TITLE
Finishing touches for S\" before issuing a pull request

### ISSUE THREAD

**Issue by SamCoVT** [SamCoVT/TaliForth2#1]
1. Mark S" (which does almost all of the work) as NN because it got so large it now has jmp instructions.
2. Write some tests for S\" specifically.  Scot's `HEXSTORE` word might be super handy here, as I could then just do a string comparison (and test both words).  Otherwise, I can just compile a string byte by byte with `C,` and use that to compare.  This will also be good testing for compare as we'll have some non-printable and control characters (like \z for a null character) in the mix.

**Comment by scotws** [comment]
`HEXSTORE` is now in master, hope that helps.

**Comment by SamCoVT** [comment]
I started using hexstore, but then decided against it because it is a Tali specific word and I was using it in string.fs.  I didn't see anything else non-ans in the string tests, so I just created a results string and character-compiled the desired bytes into it.  My thinking was that it would be better to keep the ans words tested using only ans words, to make the tests more portable.  I'm not aware of anyone else writing an ANS 2012 forth for the 65c02, but you never know.
